src/components/Numeri/components/DashboardIntro.tsx

```diff
@@ -2,8 +2,10 @@ import { Box, Stack, Typography } from "@mui/material";
 import lastUpdateSpec from "../assets/data/last-update.vl.json";
 import { toVegaLiteSpec } from "../shared/toVegaLiteSpec";
 import KpiWrapper from "./KpiWrapper";
+import { useTranslation } from "src/hook/useTranslation";
```

Comment:
Search "src/ into the all code and replace it with relative path

Reply:
ok!